featurebyte/service/online_serving.py

```diff
@@ -319,6 +319,22 @@ def _lookup_parent_entities_by_feast(
                 entity_info.format_missing_entities_error(exc.missing_entity_ids)
             )
 
+        # Validate that there are existing entity lookup tables that support the required lookup
+        # steps
+        feature_view_names = {
+            feature_view.name
+            for feature_view in feast_store._list_feature_views(  # pylint: disable=protected-access
+                allow_cache=False, hide_dummy_entity=False
+            )
+        }
+        for lookup_step in lookup_steps:
+            if get_lookup_feature_table_name(lookup_step.id) not in feature_view_names:
+                raise RequiredEntityNotProvidedError(
+                    entity_info.format_missing_entities_error(
+                        [entity.id for entity in entity_info.missing_entities]
+                    )
+                )
```

Comment:
Curious that we don't block user to get the online serving results if the entity are the child entity of the context primary entity and the lookup table is available, right?